### PR TITLE
Bug fix Image Transition improve

### LINKED ISSUE #71: Transition images properly
Transiton only between images. Not between cardview and image. (Main Activity to CharacterDetaillActivity)

### PR DESCRIPTION
**Pull request by joseantoniocampillo**
ISSUE
#71

SUMMARY
Make transition between images from MainActivity to DetaliActivity

### CODE REVIEW

File: app/src/main/java/com/architectcoders/arquitectomarvel/ui/main/pagination/CharacterAdapter.kt [hunk 1]
```diff
@@ -31,7 +31,7 @@ class CharacterAdapter(private val listener: (Result, View) -> Unit) :
         getItem(position)?.let { characterEntity ->
             holder.bind(characterEntity.toDomainCharacter)
             holder.itemView.setOnClickListener {
-                listener(characterEntity.toDomainCharacter, it)
+                listener(characterEntity.toDomainCharacter, holder.imageViewHero)
```
Comment (juancoob) [inline app/src/main/java/com/architectcoders/arquitectomarvel/ui/main/pagination/CharacterAdapter.kt:34]: Thanks @joseantoniocampillo could you change the hero word by character?

**Review by juancoob — APPROVED** [review]
@joseantoniocampillo thank you for fixing this!

File: app/src/main/java/com/architectcoders/arquitectomarvel/ui/main/pagination/CharacterAdapter.kt (hunk 1 above)
Comment (joseantoniocampillo) [reply app/src/main/java/com/architectcoders/arquitectomarvel/ui/main/pagination/CharacterAdapter.kt:34]: In the xml file, the id of ImageView referenced is hero_image. Have you seen it?

Comment (juancoob) [reply app/src/main/java/com/architectcoders/arquitectomarvel/ui/main/pagination/CharacterAdapter.kt:34]: Yeah, that should be changed too. Initially, we had some hero/heroes name, but that was changed to character in the project

**Review by JorgeGreek — APPROVED** [review]
good.

**Pull request merged**